packages/core/rsc/__snapshots__/streamable.ui.test.tsx.snap

```diff
@@ -9,13 +9,132 @@ exports[`rsc - render() > should emit React Nodes with async render function 1`]
+          "next": {
+            "done": true,
+            "value": <div
+              className="space-y-4"
+            >
+              <React.Fragment>
+                <React.Suspense>
+                  <Unknown
+                    n={
+                      {
+                        "done": false,
+                        "next": {
+                          "done": true,
+                          "value": <div>
+                            Flight 
+                            42
+                          </div>,
+                        },
+                        "value": <div>
+                          Flight 
+                          42
+                        </div>,
+                      }
+                    }
+                  />
+                </React.Suspense>
+              </React.Fragment>
+              <React.Fragment>
+                <React.Suspense>
+                  <Unknown
+                    n={
+                      {
+                        "done": false,
+                        "next": {
+                          "done": true,
+                          "value": <div>
+                            Flight 
+                            43
+                          </div>,
+                        },
+                        "value": <div>
+                          Flight 
+                          43
+                        </div>,
+                      }
+                    }
+                  />
+                </React.Suspense>
+              </React.Fragment>
+            </div>,
+          },
+          "value": <div
+            className="space-y-4"
+          >
+            <React.Fragment>
+              <React.Suspense>
+                <Unknown
+                  n={
+                    {
+                      "done": false,
+                      "next": {
+                        "done": true,
+                        "value": <div>
+                          Flight 
+                          42
+                        </div>,
+                      },
+                      "value": <div>
+                        Flight 
+                        42
+                      </div>,
+                    }
+                  }
+                />
+              </React.Suspense>
+            </React.Fragment>
+            <React.Fragment>
+              <React.Suspense>
+                <Unknown
+                  n={
+                    {
+                      "done": false,
+                      "next": {
+                        "done": true,
+                        "value": <div>
+                          Flight 
+                          43
+                        </div>,
+                      },
+                      "value": <div>
+                        Flight 
+                        43
+                      </div>,
+                    }
+                  }
+                />
+              </React.Suspense>
+            </React.Fragment>
```

Comment:
Are these snapshots now so verbose because of this?
https://github.com/vercel/ai/blob/1c5a39e0d38902c86dd0748f14b3a294b7ebfaf6/packages/core/rsc/streamable.ui.test.tsx#L106-L107